src/loaders/lottie/tvgLottieLoader.cpp

```diff
@@ -262,7 +262,8 @@ bool LottieLoader::read()
 
 Paint* LottieLoader::paint()
 {
-    this->done();
+    done();
```

Comment:
this can be changed in many other places in this file, but out of the scope of this pr